redaxo/bin/console

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env php
+<?php
+
+use Symfony\Component\Console\Input\ArgvInput;
+
+set_time_limit(0);
+
+unset($REX);
+$REX['REDAXO'] = true;
+$REX['HTDOCS_PATH'] = '../';
+$REX['BACKEND_FOLDER'] = 'redaxo';
+
+chdir(dirname(__DIR__));
+
+// initialize core
+require __DIR__.'/../src/core/boot.php';
+
+// force debug mode to enable output of notices/warnings and dump() function
+rex::setProperty('debug', true);
+
+$application = new rex_console_application();
+
+rex::setProperty('console', $application);
+
+// initialize packages
+require rex_path::core('packages.php');
+
+$input = new ArgvInput();
+
+$application->run($input);
```

Comment:
jepp, wir sollten diese datei hier auf
```
<?php
unset($REX);
$REX['REDAXO'] = true;
$REX['HTDOCS_PATH'] = '../';

require __DIR__.'/../src/core/console.php';

$REX['BACKEND_FOLDER'] = 'redaxo';
```

reduzieren und den rest in der src/core/console.php dann erledigen